mods/taskbar-thumbnail-hover-fix.wh.cpp

```diff
@@ -61,6 +79,41 @@ bool IsCursorOverThumbnail() {
         root = hwnd;
     }
 
+    // First time we encounter a window destruction, verify we're in the taskbar owner process
+    if (g_taskbarThreadId == 0) {
+        DWORD taskbarThreadId = FindTaskbarThreadId();

```

Comment:
Minor optional notes for the future:

* `FindTaskbarThreadId` could easily return the process ID too, which would make it unnecessary to use `OpenThread` below.
* You could also use `FindTaskbarThreadId` on mod init to exit early in case the taskbar is already owned by an existing process.